spec/models/article_spec.rb

```diff
@@ -544,5 +544,34 @@ def build_and_validate_article(*args)
     end
   end
 
+  describe "when algolia auto-indexing/removal is triggered" do
+    before do
+      # a default stub needed bc many triggers might happen before the tested one
+      allow(AlgoliaSearch::AlgoliaJob).to receive(:perform_later)
+    end
+
+    context "when article is saved" do
+      it "process background auto-indexing" do
```

Comment:
What do you think about using ` have_enqueued_job` helper for specs instead of stubbing a method and checking that the object has received it?

Reply:
I just love when it is over complicated :). Just kidding, you are right, so I have refactored specs. Also sorry for the delay(ah ah), I had to re install my whole system.

Reply:
:smile: no problem! Thanks for the update.